### ISSUE TITLE
`<source src=>` is not identified for resource detection 

### ISSUE THREAD

**Issue by cderv** [quarto-dev/quarto-cli#5268]
Context from https://community.rstudio.com/t/embed-sound-player-in-quarto-book/164739

````markdown
```{r}
html_tag_audio <- function(file, type = c("wav")) {
  type <- match.arg(type)
  htmltools::tags$audio(
    controls = "",
    htmltools::tags$source(
      src = file,
      type = glue::glue("audio/{type}", type = type)
    )
  )
}
```

`r html_tag_audio("t-rex-roar.mp3", type = "wav")`
````

will create something like
````html
<audio controls="">
  <source src="t-rex-roar.mp3" type="audio/wav"/>
</audio>
````

and [site-resources](https://quarto.org/docs/websites/website-tools.html#site-resources) is not detected and copied to `output-dir`

https://github.com/quarto-dev/quarto-cli/blob/75736ef241aefcd1aaa55f116bf7c40ac18a932c/src/core/html.ts#L89-L97

We should probably also find `<source src=>` for local file

**Comment by allenmanning** [comment]
Maybe related: https://github.com/quarto-dev/quarto-cli/issues/3892

**Comment by cderv** [comment]
fixed by [eb969b2](https://github.com/quarto-dev/quarto-cli/commit/eb969b2b0625d785ec57665e65129bf1c9dbde36)
